sklearn/linear_model/_bayes.py

```diff
@@ -495,9 +496,16 @@ class ARDRegression(RegressorMixin, LinearModel):
     compute_score : bool, default=False
         If True, compute the objective function at each step of the model.
 
-    threshold_lambda : float, default=10 000
-        Threshold for removing (pruning) weights with high precision from
-        the computation.
+    min_significance : float, default=0.5
+        Minimum statistical significance (|beta|/sigma) required to keep a feature.
+        Default of 0.5 provides a reasonable balance between feature selection
+        and model accuracy.
+        This replaces the threshold_lambda parameter for more interpretable feature
+        pruning.
+
+    standardize : bool, default=True
+        Whether to standardize features before fitting. Recommended for
+        consistent feature selection behavior regardless of feature scales.
```

Comment:
this needs a lot more careful consideration. We don't simply remove constructor arguments.